rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1138,6 +1138,30 @@ void save_savePendingFederation_preRSKIP123_shouldBeSavedInStorageSerializedFrom
 
     }
 
+    @Test
+    void saveActiveFederationCreationBlockHeight_shouldSaveToStorage_whenHeightIsNotNullAndRSKIP186IsActive() {
+        // Arrange
+        StorageAccessor storageAccessor = new InMemoryStorage();
```

Comment:
nit: i'd move this line right before using it when declaring the federationStorageProvider

Reply:
I wonder how that happened 🤔🤣

Reply:
Updated.